sources/osg/Polytope.js

```diff
@@ -0,0 +1,356 @@
+define( [
+    'osg/Matrix',
+    'osg/Notify',
+    'osg/Plane',
+    'osg/Utils'
+], function ( Matrix, Notify, Plane, MACROUTILS ) {
+    /*jshint bitwise: false */
+    'use strict';
+    /**
+     * Polytope class for representing convex clipping volumes made up of a set of planes.
+     * When adding planes, their normals should point inwards (into the volume)
+     * @class Polytope
+     */
+    var Polytope = function () {
+
+        this._clippingMask = 0x0;
+
+        this._planeList = [ Plane.create(), Plane.create(), Plane.create(), Plane.create(), Plane.create(), Plane.create() ];
```

Comment:
why not using new ? I remember that .create of those class is deprecated ( not sure )